Sources/StreamChatSwiftUI/ChatChannel/ChatChannelViewModel.swift

```diff
@@ -198,7 +208,9 @@ open class ChatChannelViewModel: ObservableObject, MessagesDataSource {
     @objc
     private func applicationWillEnterForeground() {
         guard let first = messages.first else { return }
-        maybeSendReadEvent(for: first)
+        if canMarkRead {
+            maybeSendReadEvent(for: first)
```

Comment:
Also, what do you think about renaming `maybeSendReadEvent()` and `maybeRefreshMessageList()` to `sendReadEventIfNeeded()` and `refreshMessageListIfNeeded()` accordingly?

Reply:
done